src/app/auth/components/createAccountForm/CreateAccountForm.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import './createAccountForm.scss';
+import { useHistory } from 'react-router-dom';
+
+import RegistrationNav from '../registrationNav';
+
+const CreateAccountForm = () => {
+  let history = useHistory();
+
+  const handleNextStep = () => {
+    history.push('/registration/upload-image');
+  };
+
+  return (
+    <>
+      <RegistrationNav title="Crea la cuenta de tu mascota." />
+
+      <form>
+        <div className="form-control">
+          <label className="form__label">Nombre de tu mascota</label>
+          <input placeholder="ej. Tommy" className="pets-input" />
+        </div>
```

Comment:
Maybe you can improve this moving the Inputs to one component in the share folder?